File: LeetCode/0944.Delete-Columns-to-Make-Sorted.cpp
// 944. Delete Columns to Make Sorted

class Solution {
public:
    int minDeletionSize(vector<string>& strs) {
        int n = strs.size();
        int len = strs[0].length();
        int ans = 0;
        for(int col=0; col<len; col++) {
            for(int row=0; row<n-1; row++) {
                if(strs[row][col] > strs[row+1][col]) {
                    ans++;
                    break;
                }
            }
        }
        return ans;
    }
};

File: LeetCode/0944-Delete-Columns-to-Make-Sorted.java
// 944. Delete Columns to Make Sorted

class Solution {
    public int minDeletionSize(String[] strs) {
        int n = strs.length, ans = 0;
        for (int j = 0; j < strs[0].length(); j++) {
            for (int i = 1; i < n; i++) {
                if (strs[i].charAt(j) < strs[i - 1].charAt(j)) {
                    ans++;
                    break;
                }
            }
        }
        return ans;
    }
}
